core/trino-main/src/main/java/io/trino/connector/system/TableCommentSystemTable.java

```diff
@@ -96,22 +103,22 @@ public RecordCursor cursor(ConnectorTransactionHandle transactionHandle, Connect
             QualifiedTablePrefix prefix = tablePrefix(catalog, schemaFilter, tableFilter);
 
             Set<SchemaTableName> names = ImmutableSet.of();
+            Map<SchemaTableName, ConnectorViewDefinition> views = ImmutableMap.of();
+            Map<SchemaTableName, ConnectorMaterializedViewDefinition> materializedViews = ImmutableMap.of();
             try {
-                names = listTables(session, metadata, accessControl, prefix);
+                materializedViews = getMaterializedViews(session, metadata, accessControl, prefix);
+                views = getViews(session, metadata, accessControl, prefix);
+                names = union(listTables(session, metadata, accessControl, prefix), views.keySet());
```

Comment:
why names union contains tables and views, but not mviews?

Reply:
I had to use a union of views and tables because some connectors like accumulo are not listing views in listTables.
For MVs, so far we are consistent in listing it as part of listTables

Reply:
can you please add a comment then, explaninig that this apparent code inconsistency is intentional and correct?

Reply:
added a comment now